src/components/PersonalInfoForm.css

```diff
@@ -0,0 +1,50 @@
+.personal-form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 20px;
+  padding: 10px;
+}
+
+.personal-form {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  align-items: center;
+}
+
+.form-item {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.phone-form-item {
+  display: flex;
+  width: 100%;
+  /* justify-content: space-around; */
```

Comment:
Is this commented and kept around for a reason?

Reply:
Removed in new commit